src/MSBuild/MSBuild.csproj

```diff
@@ -18,6 +18,7 @@
     <RootNamespace>Microsoft.Build.CommandLine</RootNamespace>
     <AssemblyName>MSBuild</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UseVSHostingProcess>false</UseVSHostingProcess>
```

Comment:
This turns off the annoying .vshost.exe process that we never need and that only gets in the way. It is never useful.